coding-challenges.js

```diff
@@ -29,17 +38,44 @@ var cubeAndSum2 = [0, 5, 10]
 
 
 
-
-
 var nums1 = [3, 56, 90, -8, 0, 23, 6]
 // Expected output: [-8, 90]
 var nums2 = [109, 5, 9, -59, 8, 24]
 // Expected output: [-59, 109]
 
+// const minMax = (array) => {
+//     let newArray = array.sort((a, b) => a-b)
+//     newArray.splice(1, 5)
+    
+//     return newArray
+// }
```

Comment:
this code doesn't work for `nums2` because you have `newArray.splice(1, 5)` hard-coded with 1 and 5. how can these numbers be written for more re-usability?